app/views/slides/show.html.erb

```diff
@@ -61,4 +61,4 @@
   </div>
 
 </div>
-<%= render 'send_to_sign', slide: @slide, signs: Sign.order(:name) %>
\ No newline at end of file
+<%= render 'send_to_sign', slide: @slide, signs: @signs %>
```

Comment:
Is it no longer necessary to use `Sign.order` due to the `before_action :set_signs`?

Reply:
Yes, because if you look at `set_signs`, it already returns the signs ordered by name.

All those `before_action` in the controller just mean before rendering the specified CRUD actions and views, run that method. You'll find the methods at the bottom of the controller.